Plugins/Dummy/CMakeLists.txt

```diff
@@ -1,49 +1,32 @@
 project(DummyPlugin)
 
 cmake_minimum_required(VERSION 2.8.11)
+if (APPLE)
+    cmake_policy(SET CMP0042 NEW)
+endif(APPLE)
```

Comment:
What is this cmake_policy for ? It would be nice to have a little  comment explaining why.